src/applications/claims-status/components/claim-overview-tab/ClaimPhaseStepper.jsx

```diff
@@ -94,6 +95,76 @@ export default function ClaimPhaseStepper({
         </>
       ),
     },
+    {
+      phase: 5,
+      header: 'Step 5: Rating',
+      description: (
+        <>
+          <p>We’ll decide your claim and determine your disability rating.</p>
+          <p>
+            If we need more evidence or you submit more evidence, your claim
+            will go back to Step 3: Evidence gathering.
+          </p>
+        </>
+      ),
+    },
+    {
+      phase: 6,
+      header: 'Step 6: Preparing decision letter',
+      description: (
+        <>
+          <p>We’ll prepare your decision letter.</p>
+          <p>
+            If you’re eligible for disability benefits, this letter will include
+            your disability rating, the amount of your monthly payments, and the
+            date your payments will start.
+          </p>
+          <p>
+            If we need more evidence or you submit more evidence, your claim
+            will go back to Step 3: Evidence gathering.
+          </p>
+        </>
+      ),
+    },
+    {
+      phase: 7,
+      header: 'Step 7: Final review',
+      description: (
+        <>
+          <p>
+            A senior reviewer will do a final review of your claim and the
+            decision letter.
+          </p>
+          <p>
+            If we need more evidence or you submit more evidence, your claim
+            will go back to Step 3: Evidence gathering.
```

Comment:
This isn't so much feedback on the pull request as it is a question re: the design. The language here is for step 7, and says that you can go back to Step 3 potentially. However, in the designs I've seen, the diagram shows that you can only go back to Step 3 between steps 4 - 6, not 7. @pmclaren19 @uxgary 

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/105806934/2d783d5d-4bdc-484a-9c5e-e6c81d777310)

Reply:
https://dsva.slack.com/archives/C056854SVT8/p1717168745828499?thread_ts=1717166351.850769&cid=C056854SVT8 @samcoforma Sounds like after somme workshops we learned the claim can revert to step 3 at any step. @uxgary is going to create a new ticket to update the diagram but we can leave this language as is!